.gitignore

```diff
@@ -4,3 +4,5 @@ node_modules
 # Don't check in build artifacts, this is only published to npm
 dist/
 
+# Don't check in new buids of test
```

Comment:
I need to check this in so that the online demo works. I'll add an .npmignore to prevent publishing that directory.